src/gui/renderer/style.ts

```diff
@@ -0,0 +1,11 @@
+export type TransitionState = 'entering' | 'entered' | 'exiting' | 'exited';
```

Comment:
I wish this was actually part of the type definitions for react-transition-group, but it's not, alas.